packages/e2e-tests/specs/experiments/blocks/navigation.test.js

```diff
@@ -595,6 +595,53 @@ describe( 'Navigation', () => {
 		expect( await getEditedPostContent() ).toMatchSnapshot();
 	} );
 
+	it( 'allows navigation submenus to open on click instead of hover', async () => {
+		await mockAllMenusResponses();
+
+		// Add the navigation block.
+		await insertBlock( 'Navigation' );
+
+		await selectDropDownOption( 'Test Menu 2' );
```

Comment:
Not added here, but my OCD was triggered by 'DropDown' 😆

Reply:
Ugh yeah, would be good to rename that if possible.